tests/cli/test_felix.py

```diff
@@ -99,6 +99,46 @@ def request_felix_landing_page(_result):
         response = test_client.post('/register', data={'address': felix.blockchain.interface.w3.eth.accounts[-1]})
         assert response.status_code == 200
 
+        # register a recipient by posting json
+        response = test_client.post(
+            '/register',
+            data=json.dumps({'address': felix.blockchain.interface.w3.eth.accounts[-2]}),
```

Comment:
If you want, you can also refer to those addresses with `felix.blockchain.unassigned_accounts`, If it's using a `TesterBlockchain`.